I can confirm that for the version of pysabnzbd that home assistant uses (1.0.1 from pypi) that providing no web root causes an API error. Trivial case:

```
test = pysabnzbd.SabnzbdApi("...", "...")
await test.check_available()

Traceback (most recent call last):
  File "/Users/arlyon/Programming/testbench/wargames/venv/lib/python3.6/site-packages/pysabnzbd/__init__.py", line 19, in call
    resp = await session.get(self.api_url, params=api_params)
  File "/Users/arlyon/Programming/testbench/wargames/venv/lib/python3.6/site-packages/aiohttp/client.py", line 357, in _request
    ssl=ssl, proxy_headers=proxy_headers, traces=traces)
  File "/Users/arlyon/Programming/testbench/wargames/venv/lib/python3.6/site-packages/aiohttp/client_reqrep.py", line 223, in __init__
    self.update_host(url)
  File "/Users/arlyon/Programming/testbench/wargames/venv/lib/python3.6/site-packages/aiohttp/client_reqrep.py", line 274, in update_host
    raise InvalidURL(url)
aiohttp.client_exceptions.InvalidURL: ...//api?apikey=...&output=json&mode=queue
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/Users/arlyon/Programming/testbench/wargames/venv/lib/python3.6/site-packages/IPython/core/interactiveshell.py", line 3261, in run_code
    last_expr = (yield from self._async_exec(code_obj, self.user_ns))
  File "cell_name", line 4, in async-def-wrapper
  File "/Users/arlyon/Programming/testbench/wargames/venv/lib/python3.6/site-packages/pysabnzbd/__init__.py", line 61, in check_available
    await self.call(params)
  File "/Users/arlyon/Programming/testbench/wargames/venv/lib/python3.6/site-packages/pysabnzbd/__init__.py", line 26, in call
    raise SabnzbdApiException('Unable to communicate with Sabnzbd API')
pysabnzbd.SabnzbdApiException: Unable to communicate with Sabnzbd API
```

That is using the latest version of sab, homeassistant, etc. pulled this week.

--- a/homeassistant/components/sabnzbd.py
+++ b/homeassistant/components/sabnzbd.py
@@ -19,7 +19,7 @@ from homeassistant.helpers.dispatcher import async_dispatcher_send
 from homeassistant.helpers.event import async_track_time_interval
 from homeassistant.util.json import load_json, save_json
 
-REQUIREMENTS = ['pysabnzbd==1.0.1']
+REQUIREMENTS = ['pysabnzbd==1.1.0']
 
 _LOGGER = logging.getLogger(__name__)
 

--- a/homeassistant/components/sabnzbd.py
+++ b/homeassistant/components/sabnzbd.py
@@ -15,6 +15,7 @@ from homeassistant.const import (
     CONF_HOST, CONF_API_KEY, CONF_NAME, CONF_PORT, CONF_SENSORS, CONF_SSL)
 from homeassistant.core import callback
 from homeassistant.helpers import discovery
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
 from homeassistant.helpers.dispatcher import async_dispatcher_send
 from homeassistant.helpers.event import async_track_time_interval
 from homeassistant.util.json import load_json, save_json
@@ -102,7 +103,8 @@ async def async_configure_sabnzbd(hass, config, use_ssl, name=DEFAULT_NAME,
                                         hass.config.path(CONFIG_FILE))
         api_key = conf.get(base_url, {}).get(CONF_API_KEY, '')
 
-    sab_api = SabnzbdApi(base_url, api_key)
+    sab_api = SabnzbdApi(base_url, api_key,
+                         session=async_get_clientsession(hass))
     if await async_check_sabnzbd(sab_api):
         async_setup_sabnzbd(hass, sab_api, config, name)
     else:
@@ -188,7 +190,8 @@ def async_request_configuration(hass, config, host):
     async def async_configuration_callback(data):
         """Handle configuration changes."""
         api_key = data.get(CONF_API_KEY)
-        sab_api = SabnzbdApi(host, api_key)
+        sab_api = SabnzbdApi(host, api_key,
+                             session=async_get_clientsession(hass))
         if not await async_check_sabnzbd(sab_api):
             return
 

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -1046,7 +1046,7 @@ pyqwikswitch==0.8
 pyrainbird==0.1.6
 
 # homeassistant.components.sabnzbd
-pysabnzbd==1.0.1
+pysabnzbd==1.1.0
 
 # homeassistant.components.climate.sensibo
 pysensibo==1.0.3

